services/movie.py

```diff
@@ -0,0 +1,33 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,

```

Comment:
It accepts Optional[list[int]] because None as default. Fix in all palces with all types

Reply:
![image](https://github.com/mate-academy/py-movie-moviesession-cinemahall/assets/121878744/d5ed556a-91dd-4fc9-9aae-2ebc593f53a2)
With Optional test issues